tests/docker_scripts/process_functional_tests_result.py

```diff
@@ -161,6 +164,24 @@ def process_result(result_path, broken_tests):
             retries,
             test_results,
         ) = process_test_log(result_path, broken_tests)
+
+        # Check test_results.tsv for sanitizer asserts, crashes and other critical errors
+        if test_results_path and os.path.exists(test_results_path):
+            with open(test_results_path, "r", encoding="utf-8") as test_results_file:
```

Comment:
Do we already have `test_results.tsv` at this point? AFAIK, `https://github.com/ClickHouse/ClickHouse/blob/9b55180184c0f38cf88bf5902687603cf903ee40/tests/docker_scripts/process_functional_tests_result.py#L222` parses run.log in order to generate and write `test_results.tsv`:
https://github.com/ClickHouse/ClickHouse/blob/9b55180184c0f38cf88bf5902687603cf903ee40/tests/docker_scripts/process_functional_tests_result.py#L222

Reply:
I reordered lines in `stateless_runner.sh`, and `test_results.tsv` is generated by `check_logs_for_critical_errors` from `stress_tests.lib`, then it is used in `process_functional_tests_result.py` and the whole `test_results.tsv` is regenerated

Reply:
Ah, ok, this was confusing, please add a comment. Does it make sense to add `--in-results-file` to make it more explicit?

Reply:
Elaborated a bit on it in the comment, added the argument as well